minecraft_servers/zeenmc/manifest.json

```diff
@@ -0,0 +1,39 @@
+{
+    "server_name": "zeenmc",
+    "nice_name": "ZeenMC Network",
+    "direct_ip": "play.zeenmc.es",
+    "server_wildcards": [
+      "%.zeenmc.com"
```

Comment:
The domain is not even registered
![image](https://github.com/LabyMod/server-media/assets/37186532/36f08684-1922-4ba9-ab38-ff0b84f64c28)
![image](https://github.com/LabyMod/server-media/assets/37186532/39b466a0-e1be-4774-a69a-a4a9715305df)